packages/integration-karma/test/rendering/slot/x/container/container.js

```diff
@@ -0,0 +1,22 @@
+import { LightningElement, api } from 'lwc';
+
+export default class Container extends LightningElement {
+    counter = 0;
+    defaultSlotCounter = 0;
+    namedSlotCounter = 0;
+
+    @api
+    reRenderElementInShadow() {
+        this.counter++;
+    }
+
+    @api
+    updateElementInDefaultSlot() {
+        this.defaultSlotCounter++;
```

Comment:
@caridy @ekashida , this triggers the slotchange event, is that correct?

Reply:
dismiss my question, i got confused. This does not triggers the slotchange event.